build/scripts/install_externals.py

```diff
@@ -0,0 +1,207 @@
+import os
+import json
+import requests
+import zipfile
+import shutil
+from enum import Enum
+
+class Host(Enum):
+    UNKNOWN = 0
+    GITLAB = 1
+    GITHUB = 2
+
+def __get_script_path():
+    return os.path.dirname(os.path.realpath(__file__))
+def __get_root_path():
+    root = __get_script_path()
+    root = os.path.join(root, os.path.join("..", ".."))
```

Comment:
look for the build folder in parent, don't use relative paths

Reply:
We will use rexpy for this which will be installed using pip